src/nvlink/wrapper.py

```
@@ -0,0 +1,1158 @@
+                    #    -> Use latin-1 to extract bytes directly
+                    #
+                    # 2. Some bytes interpreted as CP1252 by Windows/pywin32
+                    #    -> Bytes 0x80-0x9F become Unicode chars like U+201C
+                    #    -> Need to convert these back to original bytes
+                    #    -> Use module-level CP1252_TO_BYTE mapping
+                    #
+                    # 3. Proper Unicode (Japanese chars as U+3000+)
+                    #    -> Encode to cp932 for parsers
+
+                    # 高速変換: 3段階のエンコード戦略
+                    # 1. Latin-1（ASCII + 拡張ASCII）- 最速
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)